docs/source/web.md

```diff
@@ -0,0 +1,9 @@
+# Web Server
```

Comment:
Same remark as above - I think it's best to have section name identical to the file name, to the possible extent.